packages/theme-a11y/__tests__/theme-a11y.test.js

```diff
@@ -144,3 +149,125 @@ describe('bindInPageLinks()', () => {
     expect(document.activeElement).toBe(title);
   });
 });
+
+describe('accessibleLinks()', () => {
+  test('is a function exported by theme-a11y.js', () => {
+    expect(typeof accessibleLinks).toBe('function');
+  });
+
+  test('outputs a HTML list of accessible messages', () => {
+    document.body.innerHTML = `<a href="http://www.google.ca">Click me</a>`;
+
+    accessibleLinks(document.querySelectorAll('a'));
+    const messagesOutput = document.querySelectorAll('li');
+    expect(messagesOutput).toHaveLength(3);
+  });
+
+  test('overwrite default messages if parameters are provided', () => {
+    document.body.innerHTML = `
+      <a href="http://www.google.ca">Click me</a>
+      <a href="/collections/shoes">Shoes</a>
+    `;
+
+    const messages = {
+      newWindow: 'the following link will open in a new window',
```

Comment:
This is just my take but we can probably use the [sentences](https://github.com/Shopify/debut/blob/89a27af75c89acd2ffca9929390620516c33c875/src/scripts/slate/a11y.js#L180) used in Debut. They're less verbose, and given that the screen reader will already tell the user that it's a link, using the shorter sentences, e.g., "Opens in a new window" should be enough.